server_tests/test/clientTests.js

```diff
@@ -308,7 +317,7 @@ describe("RUST API TEST SUITE", function() {
     });
 
     // CLOSE SESSION
-    describe("Testing /osc/commands/execute camera.closeSession endpoint", function() {
+    describe("Testing /osc/commands/execute camera.closeSession endpoint. Disabled for testing over Wifi", function() {
```

Comment:
Er, this test in particular is not affected so this should definitely not be here. Apply the above comments to the setOptions wifiPassword test below.